src/main/java/com/salesforce/dev/framework/CommonOperation.java

```diff
@@ -0,0 +1,32 @@
+package com.salesforce.dev.framework;
+
+import org.openqa.selenium.WebDriverException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+/**
+ * Created by dante on 6/14/2016.
+ */
+public class CommonOperation {
```

Comment:
This class only will contain static methods
So, you need to have private constructor.

```java
private CommonOperation {
}
```